spring-kafka/src/test/java/org/springframework/kafka/listener/adapter/BatchMessagingMessageListenerAdapterTests.java

```diff
@@ -68,10 +112,38 @@ public static class Foo {
 
 		@KafkaListener(id = "foo", topics = "foo", autoStartup = "false")
 		public void listen(List<String> list, @Header(KafkaHeaders.GROUP_ID) String groupId) {
-			list.forEach(s -> {
-				this.value = s;
-			});
+			list.forEach(s -> this.value = s);
+			this.group = groupId;
+		}
+
+	}
+
+	public static class Bar {
```

Comment:
It is hard to require anything from you, but something like `FUBAR` is not appropriate language for professional software.
I know there are a lot of those around in the code, but I'd prefer to not have more of them from now on.

Reply:
Good point.